code/game/gamemodes/wizard/spellbook.dm

```diff
@@ -37,9 +37,9 @@
 		dat += "Uses remaining: [uses]<BR>"
 		dat += "<HR>"
 		dat += "<A href='byond://?src=\ref[src];spell_choice=spells'>Learn and Improve Magical Abilities</A><BR>"
-		dat += "<A href='byond://?src=\ref[src];spell_choice=artifacts'>Summon Magical Tools and Weapons</A><BR>"
-		dat += "<A href='byond://?src=\ref[src];spell_choice=challenge'>Raise The Stakes For More Power</A><BR>"
-
+		dat += "<A href='byond://?src=\ref[src];spell_choice=artifacts'>Summon Magical Tools and Weapons</A><BR>"
+		if(!ticker.mode == "raginmages") // we totally need summon guns x 200
```

Comment:
Don't you need an extra pair of parenthesis here? `if(!(ticker.mode == "raginmages"))`